test/hoverable.test.js

```diff
@@ -44,7 +44,8 @@ describe('o-hoverable', function() {
 		testHoverable = new oHoverable();
 		testHoverable.setClassName('hover-test');
 		expect(testHoverable.isHoverEnabled()).to.be(false);
-		document.body.classList.add('hover-test');
+		document.body.classList.add('hover-test'); // support deprecated class on body
```

Comment:
The feature is required to avoid a breaking change, and I think it's appropriate to have a test for a required feature.